docs/release-notes/release-notes-0.20.1.md

```diff
@@ -32,6 +32,11 @@
   addresses](https://github.com/lightningnetwork/lnd/pull/10341) were added to
   the node announcement and `getinfo` output.
 
+* A bug in the [implementation of
```

Comment:
not sure if we need release notes, since this function was not used in the LND release anyways, just add a no-changelog label